crates/tauri/src/scope/fs.rs

```diff
@@ -470,4 +472,79 @@ mod tests {
       assert!(scope.is_allowed("C:\\home\\tauri\\anyfile"));
     }
   }
+
+  #[cfg(windows)]
+  #[test]
+  fn windows_root_paths() {
+    let scope = new_scope();
+    {
+      // UNC network path
+      scope.allow_directory("\\\\localhost\\c$", true).unwrap();
+      assert!(scope.is_allowed("\\\\localhost\\c$"));
+      assert!(scope.is_allowed("\\\\localhost\\c$\\Windows"));
+      // Does not work because push_pattern adds a trailing backslash to the base
+      // pattern before "\**" is added leading to a pattern of "\\localhost\c$\\**"
+      // which does not match.
+      // assert!(scope.is_allowed("\\\\localhost\\c$\\NonExistentFile"));
```

Comment:
this case should definitely work, maybe just check that path doesn't end with `\` before appending `\**`